yogstation/code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -126,6 +126,13 @@ Made by Xhuis
 	text += "<br>"
 	to_chat(world, text)
 
+/datum/game_mode/shadowling/set_round_result()
+	..()
+	if(check_shadow_victory())
+		SSticker.mode_result = "win - shadowling has ascended"
```

Comment:
minor nitpick that doesn't really matter, can we have shadowling*s* instead of shadowling?

Reply:
I'll do any little thing you want, hun 😘